lib/sqlitex/sql_builder.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Sqlitex.SqlBuilder do
```

Comment:
Every Ecto adapter (MySQL, MS SQL, PostgreSQL, and SQLite) has had to implement their own SQL query generation due to differing features and functionality in each respective SQL dialect -- not that any of it is *significantly* different, rather they all fail to overlap just enough that, so far, there is little code reuse.  I like this module because the sort of SQL generation I have been writing for sqlite_ecto could be shifted to this module in the distant future.

Having said that, whether or not you want to merge this module comes down to the direction you would like Sqlitex to go:  It could remain a "dumb" (I don't use that term derogatorily) wrapper around esqlite, or it could become a sort-of-DSL for SQLite databases.  I think there is value in either direction.